### PR TITLE
Blimp: Remove comment out

### PR DESCRIPTION
**Pull request by muramura**
I am unclear about the valid process by commenting out the process.
For example, I could not find #17412.
I will remove the processing comment-out.

### CODE REVIEW

File: Blimp/GCS_Mavlink.cpp [hunk 1]
```diff
@@ -204,18 +141,6 @@ void GCS_MAVLINK_Blimp::send_pid_tuning()
         }
         const AP_Logger::PID_Info *pid_info = nullptr;
         switch (axes[i]) { //TODO This should probably become an acceleration controller?
-        // case PID_TUNING_ROLL:
```
Comment (MichelleRos) [inline Blimp/GCS_Mavlink.cpp:207]: As mentioned in my comment on #17412, I am currently implementing this code. If you want to remove this part from the PR then the rest looks ok.
Thanks.

**Review by MichelleRos — CHANGES_REQUESTED** [review]

**Comment by rmackay9** [comment]
If @MichelleRos is actively looking at implementing this code I think we should leave it in place.

@muramura is correct that we normally don't leave commented out code in place but I think we made an exception for Blimp so that it could be merged to master sooner.

I hope it's ok but I'm going to close this.  Thanks @muramura as always.

**Comment by MichelleRos** [comment]
@rmackay9 Apologies for the slight misunderstanding - of this PR's changes I am only using the PID tuning code (lines 207-218). 
The rest I don't mind if it gets removed - I'll be neatening up all of it soon in any case. 🙂

**Comment by rmackay9** [comment]
@muramura, OK, if you'd like to just leave the commented-out code around the PID code then we can merge.txs!

**Comment by tridge** [comment]
replaced by #17805

**Pull request closed without merging**